exporters/prometheus/exporter_test.go

```diff
@@ -431,6 +432,36 @@ func TestPrometheusExporter(t *testing.T) {
 				counter.Add(ctx, 5, otelmetric.WithAttributeSet(attrs2))
 			},
 		},
+		{
+			name:         "non-monotonic sum does not add exemplars",
+			expectedFile: "testdata/monotonic_sum_with_exemplars.txt",
+			recordMetrics: func(ctx context.Context, meter otelmetric.Meter) {
```

Comment:
This test doesn't fail without your fix.  You should call Add with a sampled trace context, as described in https://github.com/open-telemetry/opentelemetry-go/pull/5912#discussion_r1812909317.  That does not rely on the trace SDK (just the API, which is OK)

Reply:
confirmed it fails without the fix now, thanks again for the help.